bootstrap/command/test

```diff
@@ -18,9 +18,11 @@
 
 function is_already_formatted() {
   # FIXME: We should update how to check return value.
+  # and result is empty in appveyer so we should take care of it.
   result=$($(bootstrap_command_path)/format --diff)
   if [ "$result" = "no modified files to format" ] \
-      || [ "$result" = "clang-format did not modify any files" ]; then
+      || [ "$result" = "clang-format did not modify any files" ] \
+      || [ "$result" = "" ]; then
```

Comment:
Why does the result comes as empty string? Is it only in AppVeyor?

Reply:
Could you explain this question?

Reply:
actually i don't know. it happen only AppVeyor. so it is to hard to figure out it.